### PR TITLE
Updates README and CHANGELOG

### PR DESCRIPTION
**Pull request by oliviawongnyc**
Adds detailed release steps to the repo's README per ticket [OE-746](https://jira.nypl.org/browse/OE-746).

** Closed former PR and opened new one off the merged monorepo. **

### CODE REVIEW

**Review by samanthaandrews — APPROVED** [review]
🚀

File: apps/oew/README.md [hunk 1]
```diff
@@ -245,12 +244,20 @@ npm run cucumber -- --world-parameters '{"appUrl": "https://openebooks.org/"}'
 
 When creating links using `<Link>`, you don't need to worry about whether it is for a single or multi-library route config. Write the `as` and `href` like you would if the package only supported one-library setups, and the `<Link>` will prepend `/[libraryId]` to your routes if needed.
 
-# Deploying to production with Github Actions
+# Cutting a release and deploying to production with Github Actions
```
Comment (Toxiapo) [inline apps/oew/README.md:247]: Let's fix the TOC link at the top since this is renamed.

**Review by Toxiapo — CHANGES_REQUESTED** [review]

File: apps/oew/README.md (hunk 1 above)
Comment (oliviawongnyc) [reply apps/oew/README.md:247]: Good call.

**Review by Toxiapo — APPROVED** [review]
🎉

**Pull request merged**